ui/src/app/pages/admin/workspace/basic-information.spec.tsx

```diff
@@ -0,0 +1,108 @@
+import '@testing-library/jest-dom';
+
+import * as React from 'react';
+
+import {
+  FeaturedWorkspaceCategory,
+  WorkspaceActiveStatus,
+  WorkspaceAdminApi,
+} from 'generated/fetch';
+
+import { screen } from '@testing-library/react';
+import userEvent, { UserEvent } from '@testing-library/user-event';
+import {
+  registerApiClient,
+  workspaceAdminApi,
+} from 'app/services/swagger-fetch-clients';
+import { serverConfigStore } from 'app/utils/stores';
+
+import defaultServerConfig from 'testing/default-server-config';
+import {
+  expectButtonElementDisabled,
+  expectButtonElementEnabled,
+  renderWithRouter,
+} from 'testing/react-test-helpers';
+import { WorkspaceAdminApiStub } from 'testing/stubs/workspace-admin-api-stub';
+import { workspaceStubs } from 'testing/stubs/workspaces';
+
+import { BasicInformation } from './basic-information';
+
+describe('BasicInformation', () => {
+  const activeStatus = WorkspaceActiveStatus.ACTIVE;
+  let user: UserEvent;
+  let workspace;
+  const component = () => {
+    return renderWithRouter(
+      <BasicInformation
+        {...{ workspace, activeStatus }}
+        reload={async () => Promise.resolve()}
+      />
+    );
+  };
+
+  beforeEach(() => {
+    workspace = workspaceStubs[0];
+    user = userEvent.setup();
+    serverConfigStore.set({
+      config: {
```

Comment:
Would this also work

serverConfigStore.set({
      config: defaultServerConfig
    });

Reply:
That's cleaner. I like that. Thanks.